src/protocol.h

```diff
@@ -285,6 +285,9 @@ enum ServiceFlags : uint64_t {
     // NODE_WITNESS indicates that a node can be asked for blocks and transactions including
     // witness data.
     NODE_WITNESS = (1 << 3),
+    // NODE_COMPACT_FILTERS means the node will service basic block filter requests.
+    // See BIP157 and BIP158 for details on how this is implemented.
```

Comment:
If it's likely implementation will diverge from BIP with regards to error handling, it should be hinted somewhere. Better in release notes than here ?

Reply:
Yes, I'm happy to add release notes describing the behavior. I won't do it in this branch unless I need to retouch it (to not invalidate the three ACKs).

Reply:
@jnewbery you just requested an ack from MarcoFalke so I dunno if the ACKs have been invalidated?